lnwallet/commitment.go

```diff
@@ -660,7 +660,8 @@ type AuxLeafStore interface {
 	// correspond to the passed aux blob, and pending original (unfiltered)
 	// HTLC view.
 	FetchLeavesFromView(chanState *channeldb.OpenChannel, prevBlob tlv.Blob,
-		unfilteredView *HtlcView,
+		unfilteredView *HtlcView, isOurCommit bool, ourBalance,
+		theirBalance lnwire.MilliSatoshi,
```

Comment:
Balances needed for the new sorting algo?

Reply:
Yes, for sorting, making sure we have non-dust BTC amounts for any asset amounts and for determining whether we need commitment anchor outputs (which we need to create exclusion proofs for).